.travis.yml

```diff
@@ -9,6 +9,29 @@ env:
 
 matrix:
   include:
+  - name: 'Unit tests'
+    language: cpp
+    # Define an empty before_install phase to override the default definition
+    before_install:
+    install:
+      - sudo apt-get install --quiet=2 lcov
+    before_script:
+      - cd "${TRAVIS_BUILD_DIR}/extras/test" && mkdir build && cd build
+    script:
+      - cmake ..
+      - make
+    after_success:
+      # Check code coverage
+      - lcov --directory . --zerocounters
+      - bin/test-107-Arduino-UAVCAN
+      - lcov --directory . --capture --output-file coverage.info
+      - lcov --remove coverage.info 'test/*' '/usr/*' 'libcanard/*' 'o1heap/*' --output-file coverage.info
```

Comment:
I have now added the unit tests to the CI build as requested. Following the example of ArduinoCloudThing's CI configuration, I also added the code coverage check. I don't have any prior experience with this so I'm not sure I got it right and I couldn't fully test it since there aren't any tests yet. So please take a look at this to see if you spot any problems.

Reply:
I've set everything up with `codecov.io` but it doesn't display any coverage yet. I believe this is due to the fact that there are no tests run yet :wink: Thank you for the work!